README.md

```diff
@@ -1,59 +1,104 @@
-README
-------
+OIP
+---
 
-Use libpcap and SDL to visualize IP traffic between multiple endpoints
+OrganicIP (OIP) visualizer uses libpcap and Simple DirectMedia Layer (SDL) to visualize IP traffic between endpoints.
 
-Packages
---------
- * `deb http://mirror.usu.edu/usu-security/ [codename]-security-unstable main`
-     * Debian: sid, bullseye
-     * Ubuntu: impish, focal, bionic
- * install `usu-archive-keyring` package and apt-get update
-     * keyring file available at `https://raw.githubusercontent.com/USU-Security/oip/master/usu_archive_keyring.gpg`
+Supported Distributions
```

Comment:
I'd like to make it clear that these are pre-built packages that are available.  The software is supported on any Linux system with the necessary dependencies installed, even though we don't have pre-built packages for everything.

Reply:
Added an introductory sentence to include the language you proposed. Hope this helps set the stage and better link to other locations.